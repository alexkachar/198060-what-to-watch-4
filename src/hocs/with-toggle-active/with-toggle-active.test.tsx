import * as React from 'react';
import * as renderer from 'react-test-renderer';
import withToggleActive from './with-toggle-active';

const MockComponent = () => <div />;
const WrappedComponent = withToggleActive(MockComponent);

it(`withActiveTab renders correctly`, () => {
  const tree = renderer
    .create(
        <WrappedComponent />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
