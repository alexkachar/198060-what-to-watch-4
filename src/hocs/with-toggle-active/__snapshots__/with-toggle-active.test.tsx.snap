// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveTab renders correctly 1`] = `<div />`;
